### ISSUE TITLE
1.9.0 core.matrix dependency

### ISSUE THREAD

**Issue by malloc82** [incanter/incanter#325]
1.9.0 branch is stuck with core.matrix 0.45 at the moment. After 0.45 there seems to be some API updates that breaks incanter, for example:
with core.matrix 0.46, I have this error when compiling my code, which is working with 0.45:
```
CompilerException java.lang.IllegalStateException: var: #'clojure.core.matrix.dataset/dataset? is not public, compiling:(incanter/core.clj:99:10)
```
with core.matrix 0.49 (latest as of this post), I get this error:
```
CompilerException java.lang.RuntimeException: No such var: ds/except-columns, compiling:(incanter/core.clj:1402:47) 
```

**Comment by malloc82** [comment]
I guess I need to clarify that I'm also using core.matrix in my project So it seems that it overrides incanter's 0.43 dependency, and thus the compiling errors I have here.

**Comment by mikera** [comment]
Hmmm it probably needs a new Incanter release to fix up all the potential issues

**Comment by alexott** [comment]
If it's fixed in the master, then I'm ready to push the new release to clojars - I simply don't have to fix it myself...

**Comment by alexott** [comment]
fixed via 95b41f2
